source/fab/steps/analyse.py

```diff
@@ -258,6 +256,21 @@ def _parse_files(self, files: List[Path]) -> Set[AnalysedFile]:
 
         return set(by_type(results, AnalysedFile))
 
+    def _add_manual_results(self, analysed_files: Set[AnalysedFile]):
+        # add manual analysis results for files which could not be parsed
+        if self.special_measure_analysis_results:
+            warnings.warn("SPECIAL MEASURE: injecting user-defined analysis results")
+            manual_results = set()
+            for r in self.special_measure_analysis_results:
+                if isinstance(r, ParserWorkaround):
+                    manual_results.add(r.as_analysed_file())
+                elif isinstance(r, AnalysedFile):
```

Comment:
Do we need to be backward compatible at this stage? We have a small beta user base.

Reply:
Not desperately. I thought it was the polite thing to do, since they're testing for us. Want it gone?

Reply:
I think it's time to be ruthless. We don't want to go into an initial release with baggage and technical debt pre-installed.